docs/data/sdks/typescript-browser/migration.md

```diff
@@ -0,0 +1,497 @@
+---
+title: Migration Guide
+description: This is a guide on migrating from amplitude-js
+---
+
+Amplitude has releases a new version of Browser SDK (`@amplitude/analytics-browser`), which features a plugin architecture, built-in type definition and broader support for front-end frameworks. The new version is not backwards compatibly with the `amplitude-js` version. 
+
+To migrate to `@amplitude/analytics-browser`, the dependency and instrumentation must be updated.
+
+### Terminology
+
+* `amplitude-js`: Legacy Browser SDK
+* `@amplitude/analytics-browser`: New Browser SDK
+
+## Dependency
+
+For snippet installation, update your project"s snippet loader. The new snippet loader can be found here: [https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser#using-script-loader](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser#using-script-loader)
+
+For Node projects, update your dependency list in package.json.
+
+=== "amplitude-js"
+  ```json
+  {
+    "dependencies": {
+      "amplitude-js": "8"
+    }
+  }
+  ```
+
+=== "amplitude-js"
+  ```json
+  {
+    "dependencies": {
+      "@amplitude/analytics-browser": "1"
+    }
+  }
+  ```
+
+## Instrumentation
+
+The new Browser SDK offers a new API to instrument events. To move forward with the migration, the following API calls must be changed.
+
+### Initialization
+
+Similar to all other calls, `getInstance()` has been removed. To initialize the SDK, call `init()`, with the same parameters. However, `config` comes in a different shape. See [Configuration](#configuration).
```

Comment:
Do we need to mention, that we don't support multiple instants for now?

Reply:
multi-instance support is scheduled to be worked on this week @yuhao900914